TF-Security/Charter.md

```diff
@@ -0,0 +1,80 @@
+
+# Web of Things Security Task Force Charter
+The mission of this task force at the highest level is to ensure everything that the W3C WoT WG produces is secure by default.  Incidents such as [Mirai](https://en.wikipedia.org/wiki/Mirai_(malware)) reinforce the need to ensure security up front.
+
+## Task Force Details
+
+**End Date:** Coincident with WoT WG
+
+**Confidentiality:** Proceedings are [public](https://www.w3.org/2005/10/Process-20051014/comm.html#confidentiality-levels).
+
+**Initial Chair:** Ryan Ware
+
+**Initial Team Contact:** Dave Raggett
+
+**Meeting Schedule:** TBD
+
+
+## Table of Contents
+1. [Scope](#scope)
+2. [Deliverables](#deliverables)
+3. [Relationships to External Groups](#relationships-to-external-groups)
+4. [Participation](#participation)
+5. [Communication](#communication)
+6. [About This Charter](#about-this-charter)
+
+# Scope
+The WoT Security Task Force is responsible for:
+
+* Review of WoT related specifications for specific security relevant properties.  These include areas related to:
+    * Authentication
+    * Authorization
+    * Encryption
+    * Auditing
+* Review of use cases including:
+    * Onboarding
+    * Provisioning
+    * Compromise Recovery
+* Security Test Plans for Specifications
+* Suggested Test Plans for Implementations
+* Security Compliance Test Suite
+
+# Deliverables
+The WoT Security Task Force will not produce recommendation-track deliverables but will produce Working Group Notes on security in conformance with the [scope](#scope) of this document.  This will include specification reviews, use case reviews and security test plans.
+
```

Comment:
This is somewhat at odds with the WG charter proposal, where security recommendations are cross-cutting but specific recommendations eg in the TD should be normative.   How do we generate concrete recommendations in the WG: do we need a separate Security TG in the WG, after it is chartered?